packages/react/src/components/Table/TableSaveViewModal/TableSaveViewModal.jsx

```diff
@@ -145,7 +153,8 @@ const TableSaveViewModal = ({
         isPrimaryButtonDisabled: formValues.title === '',
       }}
       iconDescription={closeIconDescription}
-      testID={testID}
+      // TODO: remove deprecated 'testID' in v3
+      testID={testID || testId}
```

Comment:
Leaving this as capital `testID` prop, because this component is overridable and the change could potentially be breaking for someone overriding the modal component with their own.